examples/lines_bars_and_markers/linestyles.py

```diff
@@ -6,8 +6,9 @@
 Simple linestyles can be defined using the strings "solid", "dotted", "dashed"
 or "dashdot". More refined control can be achieved by providing a dash tuple
 ``(offset, (on_off_seq))``. For example, ``(0, (3, 10, 1, 15))`` means
-(3pt line, 10pt space, 1pt line, 15pt space) with no offset. See also
-`.Line2D.set_linestyle`.
+(3pt line, 10pt space, 1pt line, 15pt space) with no offset, while
+``(5, (10, 5))``, means (10pt line, 5pt space), but skip the first 5pt line.
```

Comment:
The two 5s have separate effects. Maybe use different numbers to make clear which one is which?

Reply:
That indeed makes sense from a documentation perspective.